studies/forms.py

```diff
@@ -188,10 +212,11 @@ class EligibleParticipantQueryModelForm(ModelForm):
 
     gender_specification = forms.IntegerField(required=False)
 
-    # TODO: validation function
+    # # TODO: validation function
```

Comment:
actually still todo?

Reply:
Nah but I'll be deleting this form anyway - just waiting to have everything tested in dev for the query DSL. I can get rid of it now if it bothers you though :)

Reply:
Nope it's fine! Just didn't realize this was that form.